homeassistant/components/ue_smart_radio/media_player.py

```diff
@@ -82,10 +82,10 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     session = session_request.cookies["sdi_squeezenetwork_session"]
 
     player_request = send_request({"params": ["", ["serverstatus"]]}, session)
-    player_id = player_request["result"]["players_loop"][0]["playerid"]
-    player_name = player_request["result"]["players_loop"][0]["name"]
-
-    add_entities([UERadioDevice(session, player_id, player_name)])
+    for player in player_request["result"]["players_loop"]:
+        player_id = player["playerid"]
+        player_name = player["name"]
+        add_entities([UERadioDevice(session, player_id, player_name)])
```

Comment:
Why was only one device supported earlier? What's the impact of this change?

Please collect the entities in a list first then make a single call to `add_entities`.

Reply:
I'm sorry but I do not know why the original author only added the first device. If you only have one device then there is no impact of my change. When you have multiple devices they are now all added as separate media players. Each work independently but share the same session.
I have made the requested change to now make a single call to add_entities

Reply:
Ok.